internal/typeinfo/member.go

```diff
@@ -143,8 +144,9 @@ func (f *structField) Desc() string {
 	return "tag \"" + f.tag + "\" of struct \"" + f.structType.Name() + "\""
```

Comment:
This reads much nicer as `fmt.Sprintf("tag %q of struct %q", f.tag, f.structType.Name())`

Reply:
Changed.